content/guides/exporters/supported-exporters/Node.js/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Node"
+date: 2018-08-15T14:26:12-07:00
+draft: false
+weight: 2
+class: "resized-logo"
+---
+
+![](/images/nodejs.png)
+
+OpenCensus Node provides support for various exporters like:
+
+* [Google Stackdriver Tracing](/supported-exporters/node/stackdriver-trace/)
```

Comment:
All these exporter links seem dead within this repo(not yet committed). Let's instead use
```
{{% children %}}
```
so that they'll automatically be added as the respective exporter sections are added

Reply:
Yep, that's better! Thanks for the tip!